kokoro/builds/build.sh

```diff
@@ -15,6 +15,10 @@ if [ -z "${CONAN_PROFILE}" ]; then
   declare -A profile_mapping=( \
     [ggp_relwithdebinfo]=skip \
     [msvc2019_relwithdebinfo]=skip \
+    [clang7_relwithdebinfo]=skip \
+    [gcc9_relwithdebinfo]=skip \
```

Comment:
Does this mean that we keep these profiles around but don't support them?

Reply:
The profiles/compilers still work, but we can't provide a CI docker container for them with reasonable effort.

Also we keep the CI jobs around because we decided not to make any changes to the Kokoro config for now. That's why we just skip them. (They would continue to run on the stadia_main branch)